core/src/java/org/hypergraphdb/util/OnGCCheckMemoryWarningSystem.java

```diff
@@ -0,0 +1,158 @@
+/* 
```

Comment:
This file seems to contain a copy&pasted comment and it also seems like it's using tabs instead of spaces- need 4 spaces indentation, no tabs :)